[
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     768     gcc: _configtest.c\n769     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     770         3 | #pragma clang diagnostic error \"-Wattributes\"\n     771           |\n     772     _configtest.c: In function '__declspec':\n  >> 773     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "774         5 | int __declspec(thread) foo;\n     775           |                        ^~~\n  >> 776     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     777         9 | {\n     778           | ^\n779     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     780         5 | int __declspec(thread) foo;\n     781           |     ^~~~~~~~~~\n  >> 782     _configtest.c:11: error: expected '{' at end of input\n     783        11 | }\n     784           |",
    "implicit-int]\n     780         5 | int __declspec(thread) foo;\n     781           |     ^~~~~~~~~~\n  >> 782     _configtest.c:11: error: expected '{' at end of input\n     783        11 | }\n     784           |\n785     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     786        11 | }\n     787           |\n     788     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     789         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "783        11 | }\n     784           |\n     785     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     786        11 | }\n     787           |\n788     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     789         3 | #pragma clang diagnostic error \"-Wattributes\"\n     790           |\n     791     _configtest.c: In function '__declspec':\n  >> 792     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "793         5 | int __declspec(thread) foo;\n     794           |                        ^~~\n  >> 795     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     796         9 | {\n     797           | ^\n798     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     799         5 | int __declspec(thread) foo;\n     800           |     ^~~~~~~~~~\n  >> 801     _configtest.c:11: error: expected '{' at end of input\n     802        11 | }\n     803           |",
    "implicit-int]\n     799         5 | int __declspec(thread) foo;\n     800           |     ^~~~~~~~~~\n  >> 801     _configtest.c:11: error: expected '{' at end of input\n     802        11 | }\n     803           |\n804     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     805        11 | }\n     806           |\n     807     failure.\n\n     ...",
    "...\n\n     2791          |                              ^~\n     2792    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n2793    numpy/core/src/multiarray/scalartypes.c.src:2727:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2794     2727 |         if (ret->obmeta.base == -1) {\n     2795          |                              ^~\n     2796    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2816     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2817          |                              ^\n     2818    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':\n2819    numpy/core/src/multiarray/scalartypes.c.src:627:5: warning: 'PyUni\n             code_AsUnicode' is deprecated [-Wdeprecated-declarations]\n     2820      627 |     ip = dptr = Py@Name@_AS_@NAME@(self);\n     2821          |     ^~\n\n     ...",
    "...\n\n     2958          |                              ^~\n     2959    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n2960    numpy/core/src/multiarray/scalartypes.c.src:2727:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2961     2727 |         if (ret->obmeta.base == -1) {\n     2962          |                              ^~\n     2963    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "st.o -o _configtest\n     94      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 95      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 96      collect2: error: ld returned 1 exit status\n     97      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 98      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 99      collect2: error: ld returned 1 exit status",
    ">> 96      collect2: error: ld returned 1 exit status\n     97      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 98      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 99      collect2: error: ld returned 1 exit status\n     100     failure.\n     101     removing: _configtest.c _configtest.o\n     102     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     103",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     252     gcc: _configtest.c\n     253     _configtest.c: In function 'main':\n>> 254     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     255         7 |     (void) SIZEOF_LONGDOUBLE;\n     256           |            ^~~~~~~~~~~~~~~~~\n     257           |            SIZEOF_LONG_DOUBLE\n     258     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "255         7 |     (void) SIZEOF_LONGDOUBLE;\n     256           |            ^~~~~~~~~~~~~~~~~\n     257           |            SIZEOF_LONG_DOUBLE\n     258     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     259     _configtest.c: In function 'main':\n>> 260     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     261         7 |     (void) SIZEOF_LONGDOUBLE;\n     262           |            ^~~~~~~~~~~~~~~~~\n     263           |            SIZEOF_LONG_DOUBLE\n     264     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     398     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 399     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 400     collect2: error: ld returned 1 exit status\n     401     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 402     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 403     collect2: error: ld returned 1 exit status",
    ">> 400     collect2: error: ld returned 1 exit status\n     401     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 402     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 403     collect2: error: ld returned 1 exit status\n     404     failure.\n     405     removing: _configtest.c _configtest.o\n     406     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     407",
    "768     gcc: _configtest.c\n     769     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     770         3 | #pragma clang diagnostic error \"-Wattributes\"\n     771           |\n     772     _configtest.c: In function '__declspec':\n>> 773     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     774         5 | int __declspec(thread) foo;\n     775           |                        ^~~\n  >> 776     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     777         9 | {",
    "771           |\n     772     _configtest.c: In function '__declspec':\n  >> 773     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     774         5 | int __declspec(thread) foo;\n     775           |                        ^~~\n>> 776     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     777         9 | {\n     778           | ^\n     779     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     780         5 | int __declspec(thread) foo;",
    "777         9 | {\n     778           | ^\n     779     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     780         5 | int __declspec(thread) foo;\n     781           |     ^~~~~~~~~~\n>> 782     _configtest.c:11: error: expected '{' at end of input\n     783        11 | }\n     784           |\n     785     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     786        11 | }\n     787           |",
    "787           |\n     788     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     789         3 | #pragma clang diagnostic error \"-Wattributes\"\n     790           |\n     791     _configtest.c: In function '__declspec':\n>> 792     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     793         5 | int __declspec(thread) foo;\n     794           |                        ^~~\n  >> 795     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     796         9 | {",
    "790           |\n     791     _configtest.c: In function '__declspec':\n  >> 792     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     793         5 | int __declspec(thread) foo;\n     794           |                        ^~~\n>> 795     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     796         9 | {\n     797           | ^\n     798     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     799         5 | int __declspec(thread) foo;",
    "796         9 | {\n     797           | ^\n     798     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     799         5 | int __declspec(thread) foo;\n     800           |     ^~~~~~~~~~\n>> 801     _configtest.c:11: error: expected '{' at end of input\n     802        11 | }\n     803           |\n     804     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     805        11 | }\n     806           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-libensemble/spack_env_dir/linu\n             x-rhel7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd\n             3g35si6/include/python3.9 -c'\n     1134    gcc: _configtest.c\n     1135    _configtest.c: In function 'main':\n>> 1136    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1137        8 |     (void) HAVE_DECL_SIGNBIT;\n     1138          |            ^~~~~~~~~~~~~~~~~\n     1139          |            HAVE_DECL_ISNAN\n     1140    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1137        8 |     (void) HAVE_DECL_SIGNBIT;\n     1138          |            ^~~~~~~~~~~~~~~~~\n     1139          |            HAVE_DECL_ISNAN\n     1140    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1141    _configtest.c: In function 'main':\n>> 1142    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1143        8 |     (void) HAVE_DECL_SIGNBIT;\n     1144          |            ^~~~~~~~~~~~~~~~~\n     1145          |            HAVE_DECL_ISNAN\n     1146    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1724    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1725    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1726    collect2: error: ld returned 1 exit status\n     1727    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1728    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1729    collect2: error: ld returned 1 exit status",
    ">> 1726    collect2: error: ld returned 1 exit status\n     1727    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1728    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1729    collect2: error: ld returned 1 exit status\n     1730    failure.\n     1731    removing: _configtest.c _configtest.o\n     1732    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1733",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2794     2727 |         if (ret->obmeta.base == -1) {\n     2795          |                              ^~\n     2796    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 2797    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2798     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2799          |                       ^\n  >> 2800    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2801     4184 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "2796    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 2797    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2798     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2799          |                       ^\n>> 2800    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2801     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2802          |                        ^\n  >> 2803    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2804     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "2798     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2799          |                       ^\n  >> 2800    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2801     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2802          |                        ^\n>> 2803    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2804     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2805          |                         ^\n  >> 2806    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2807     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "2801     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2802          |                        ^\n  >> 2803    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2804     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2805          |                         ^\n>> 2806    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2807     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2808          |                             ^\n  >> 2809    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "2805          |                         ^\n  >> 2806    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2807     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2808          |                             ^\n>> 2809    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2810     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2811          |                         ^\n  >> 2812    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2813     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     2808          |                             ^\n  >> 2809    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2810     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2811          |                         ^\n>> 2812    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2813     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2814          |                          ^\n  >> 2815    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2816     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "2810     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2811          |                         ^\n  >> 2812    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2813     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2814          |                          ^\n>> 2815    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2816     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2817          |                              ^\n     2818    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2961     2727 |         if (ret->obmeta.base == -1) {\n     2962          |                              ^~\n     2963    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 2964    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2965     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2966          |                       ^\n  >> 2967    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2968     4184 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "2963    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 2964    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2965     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2966          |                       ^\n>> 2967    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2968     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2969          |                        ^\n  >> 2970    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2971     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "2965     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2966          |                       ^\n  >> 2967    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2968     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2969          |                        ^\n>> 2970    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2971     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2972          |                         ^\n  >> 2973    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2974     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "2968     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2969          |                        ^\n  >> 2970    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2971     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2972          |                         ^\n>> 2973    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2974     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2975          |                             ^\n  >> 2976    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "2972          |                         ^\n  >> 2973    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2974     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2975          |                             ^\n>> 2976    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2977     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2978          |                         ^\n  >> 2979    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2980     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     2975          |                             ^\n  >> 2976    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2977     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2978          |                         ^\n>> 2979    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2980     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2981          |                          ^\n  >> 2982    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2983     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "2977     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2978          |                         ^\n  >> 2979    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2980     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2981          |                          ^\n>> 2982    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2983     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2984          |                              ^\n  >> 2985    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g\n             cc/gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -",
    "y/core -Inumpy/core/src/npymath -Inumpy/core/src/multiarray -Inump\n             y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     768     gcc: _configtest.c\n769     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     770         3 | #pragma clang diagnostic error \"-Wattributes\"\n     771           |\n     772     _configtest.c: In function '__declspec':\n  >> 773     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "774         5 | int __declspec(thread) foo;\n     775           |                        ^~~\n  >> 776     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     777         9 | {\n     778           | ^\n779     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     780         5 | int __declspec(thread) foo;\n     781           |     ^~~~~~~~~~\n  >> 782     _configtest.c:11: error: expected '{' at end of input\n     783        11 | }\n     784           |",
    "implicit-int]\n     780         5 | int __declspec(thread) foo;\n     781           |     ^~~~~~~~~~\n  >> 782     _configtest.c:11: error: expected '{' at end of input\n     783        11 | }\n     784           |\n785     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     786        11 | }\n     787           |\n     788     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     789         3 | #pragma clang diagnostic error \"-Wattributes\"",
    "783        11 | }\n     784           |\n     785     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     786        11 | }\n     787           |\n788     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     789         3 | #pragma clang diagnostic error \"-Wattributes\"\n     790           |\n     791     _configtest.c: In function '__declspec':\n  >> 792     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'",
    "793         5 | int __declspec(thread) foo;\n     794           |                        ^~~\n  >> 795     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     796         9 | {\n     797           | ^\n798     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     799         5 | int __declspec(thread) foo;\n     800           |     ^~~~~~~~~~\n  >> 801     _configtest.c:11: error: expected '{' at end of input\n     802        11 | }\n     803           |",
    "implicit-int]\n     799         5 | int __declspec(thread) foo;\n     800           |     ^~~~~~~~~~\n  >> 801     _configtest.c:11: error: expected '{' at end of input\n     802        11 | }\n     803           |\n804     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     805        11 | }\n     806           |\n     807     failure.\n\n     ...",
    "...\n\n     2791          |                              ^~\n     2792    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n2793    numpy/core/src/multiarray/scalartypes.c.src:2727:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2794     2727 |         if (ret->obmeta.base == -1) {\n     2795          |                              ^~\n     2796    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2816     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2817          |                              ^\n     2818    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':\n2819    numpy/core/src/multiarray/scalartypes.c.src:627:5: warning: 'PyUni\n             code_AsUnicode' is deprecated [-Wdeprecated-declarations]\n     2820      627 |     ip = dptr = Py@Name@_AS_@NAME@(self);\n     2821          |     ^~\n\n     ...",
    "...\n\n     2958          |                              ^~\n     2959    numpy/core/src/multiarray/scalartypes.c.src: In function 'timedelt\n             a_arrtype_new':\n2960    numpy/core/src/multiarray/scalartypes.c.src:2727:30: warning: comp\n             arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2961     2727 |         if (ret->obmeta.base == -1) {\n     2962          |                              ^~\n     2963    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':",
    "st.o -o _configtest\n     94      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 95      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 96      collect2: error: ld returned 1 exit status\n     97      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 98      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 99      collect2: error: ld returned 1 exit status",
    ">> 96      collect2: error: ld returned 1 exit status\n     97      /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 98      /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 99      collect2: error: ld returned 1 exit status\n     100     failure.\n     101     removing: _configtest.c _configtest.o\n     102     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     103",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     252     gcc: _configtest.c\n     253     _configtest.c: In function 'main':\n>> 254     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     255         7 |     (void) SIZEOF_LONGDOUBLE;\n     256           |            ^~~~~~~~~~~~~~~~~\n     257           |            SIZEOF_LONG_DOUBLE\n     258     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "255         7 |     (void) SIZEOF_LONGDOUBLE;\n     256           |            ^~~~~~~~~~~~~~~~~\n     257           |            SIZEOF_LONG_DOUBLE\n     258     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     259     _configtest.c: In function 'main':\n>> 260     _configtest.c:7:12: error: 'SIZEOF_LONGDOUBLE' undeclared (first u\n             se in this function); did you mean 'SIZEOF_LONG_DOUBLE'?\n     261         7 |     (void) SIZEOF_LONGDOUBLE;\n     262           |            ^~~~~~~~~~~~~~~~~\n     263           |            SIZEOF_LONG_DOUBLE\n     264     _configtest.c:7:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     398     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 399     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 400     collect2: error: ld returned 1 exit status\n     401     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 402     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 403     collect2: error: ld returned 1 exit status",
    ">> 400     collect2: error: ld returned 1 exit status\n     401     /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 402     /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 403     collect2: error: ld returned 1 exit status\n     404     failure.\n     405     removing: _configtest.c _configtest.o\n     406     C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     407",
    "768     gcc: _configtest.c\n     769     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     770         3 | #pragma clang diagnostic error \"-Wattributes\"\n     771           |\n     772     _configtest.c: In function '__declspec':\n>> 773     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     774         5 | int __declspec(thread) foo;\n     775           |                        ^~~\n  >> 776     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     777         9 | {",
    "771           |\n     772     _configtest.c: In function '__declspec':\n  >> 773     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     774         5 | int __declspec(thread) foo;\n     775           |                        ^~~\n>> 776     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     777         9 | {\n     778           | ^\n     779     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     780         5 | int __declspec(thread) foo;",
    "777         9 | {\n     778           | ^\n     779     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     780         5 | int __declspec(thread) foo;\n     781           |     ^~~~~~~~~~\n>> 782     _configtest.c:11: error: expected '{' at end of input\n     783        11 | }\n     784           |\n     785     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     786        11 | }\n     787           |",
    "787           |\n     788     _configtest.c:3: warning: ignoring #pragma clang diagnostic [-Wunk\n             nown-pragmas]\n     789         3 | #pragma clang diagnostic error \"-Wattributes\"\n     790           |\n     791     _configtest.c: In function '__declspec':\n>> 792     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     793         5 | int __declspec(thread) foo;\n     794           |                        ^~~\n  >> 795     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     796         9 | {",
    "790           |\n     791     _configtest.c: In function '__declspec':\n  >> 792     _configtest.c:5:24: error: expected declaration specifiers before\n             'foo'\n     793         5 | int __declspec(thread) foo;\n     794           |                        ^~~\n>> 795     _configtest.c:9:1: error: expected '=', ',', ';', 'asm' or '__attr\n             ibute__' before '{' token\n     796         9 | {\n     797           | ^\n     798     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     799         5 | int __declspec(thread) foo;",
    "796         9 | {\n     797           | ^\n     798     _configtest.c:5:5: warning: type of 'thread' defaults to 'int' [-W\n             implicit-int]\n     799         5 | int __declspec(thread) foo;\n     800           |     ^~~~~~~~~~\n>> 801     _configtest.c:11: error: expected '{' at end of input\n     802        11 | }\n     803           |\n     804     _configtest.c:11: warning: control reaches end of non-void functio\n             n [-Wreturn-type]\n     805        11 | }\n     806           |",
    "y/core/src/umath -Inumpy/core/src/npysort -I/p/vast1/koparasy/BUIL\n             D/spack_cluster_installer/quartz/py-warpx/spack_env_dir/linux-rhel\n             7-broadwell/gcc-9.3.1/python-3.9.9-3o76m4utiyrsvuxwx2ytcv7sd3g35si\n             6/include/python3.9 -c'\n     1134    gcc: _configtest.c\n     1135    _configtest.c: In function 'main':\n>> 1136    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1137        8 |     (void) HAVE_DECL_SIGNBIT;\n     1138          |            ^~~~~~~~~~~~~~~~~\n     1139          |            HAVE_DECL_ISNAN\n     1140    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "1137        8 |     (void) HAVE_DECL_SIGNBIT;\n     1138          |            ^~~~~~~~~~~~~~~~~\n     1139          |            HAVE_DECL_ISNAN\n     1140    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in\n     1141    _configtest.c: In function 'main':\n>> 1142    _configtest.c:8:12: error: 'HAVE_DECL_SIGNBIT' undeclared (first u\n             se in this function); did you mean 'HAVE_DECL_ISNAN'?\n     1143        8 |     (void) HAVE_DECL_SIGNBIT;\n     1144          |            ^~~~~~~~~~~~~~~~~\n     1145          |            HAVE_DECL_ISNAN\n     1146    _configtest.c:8:12: note: each undeclared identifier is reported o\n             nly once for each function it appears in",
    "st.o -o _configtest\n     1724    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1725    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1726    collect2: error: ld returned 1 exit status\n     1727    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1728    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n  >> 1729    collect2: error: ld returned 1 exit status",
    ">> 1726    collect2: error: ld returned 1 exit status\n     1727    /usr/tce/packages/gcc/gcc-9.3.1/rh/usr/bin/../libexec/gcc/x86_64-r\n             edhat-linux/9/ld: _configtest.o: in function `main':\n  >> 1728    /tmp/koparasy/spack-stage/spack-stage-py-numpy-1.11.0-2c3v2tp7twx5\n             be3tbaqcdtdib5v6slrr/spack-src/_configtest.c:6: undefined referenc\n             e to `exp'\n>> 1729    collect2: error: ld returned 1 exit status\n     1730    failure.\n     1731    removing: _configtest.c _configtest.o\n     1732    C compiler: /p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gcc/g\n             cc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall\n              -fPIC -fPIC\n     1733",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2794     2727 |         if (ret->obmeta.base == -1) {\n     2795          |                              ^~\n     2796    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 2797    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2798     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2799          |                       ^\n  >> 2800    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2801     4184 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "2796    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 2797    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2798     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2799          |                       ^\n>> 2800    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2801     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2802          |                        ^\n  >> 2803    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2804     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "2798     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2799          |                       ^\n  >> 2800    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2801     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2802          |                        ^\n>> 2803    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2804     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2805          |                         ^\n  >> 2806    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2807     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "2801     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2802          |                        ^\n  >> 2803    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2804     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2805          |                         ^\n>> 2806    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2807     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2808          |                             ^\n  >> 2809    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "2805          |                         ^\n  >> 2806    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2807     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2808          |                             ^\n>> 2809    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2810     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2811          |                         ^\n  >> 2812    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2813     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     2808          |                             ^\n  >> 2809    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2810     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2811          |                         ^\n>> 2812    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2813     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2814          |                          ^\n  >> 2815    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2816     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "2810     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2811          |                         ^\n  >> 2812    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2813     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2814          |                          ^\n>> 2815    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2816     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2817          |                              ^\n     2818    numpy/core/src/multiarray/scalartypes.c.src: In function 'unicodet\n             ype_repr':",
    "arison of integer expressions of different signedness: 'NPY_DATETI\n             MEUNIT' {aka 'enum <anonymous>'} and 'int' [-Wsign-compare]\n     2961     2727 |         if (ret->obmeta.base == -1) {\n     2962          |                              ^~\n     2963    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n>> 2964    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2965     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2966          |                       ^\n  >> 2967    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2968     4184 |     PyFloatArrType_Type.tp_print = floattype_print;",
    "2963    numpy/core/src/multiarray/scalartypes.c.src: In function 'initiali\n             ze_numeric_types':\n  >> 2964    numpy/core/src/multiarray/scalartypes.c.src:4183:23: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2965     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2966          |                       ^\n>> 2967    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2968     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2969          |                        ^\n  >> 2970    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2971     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;",
    "2965     4183 |     PyHalfArrType_Type.tp_print = halftype_print;\n     2966          |                       ^\n  >> 2967    numpy/core/src/multiarray/scalartypes.c.src:4184:24: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2968     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2969          |                        ^\n>> 2970    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2971     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2972          |                         ^\n  >> 2973    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2974     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri",
    "2968     4184 |     PyFloatArrType_Type.tp_print = floattype_print;\n     2969          |                        ^\n  >> 2970    numpy/core/src/multiarray/scalartypes.c.src:4185:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2971     4185 |     PyDoubleArrType_Type.tp_print = doubletype_print;\n     2972          |                         ^\n>> 2973    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2974     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2975          |                             ^\n  >> 2976    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'",
    "2972          |                         ^\n  >> 2973    numpy/core/src/multiarray/scalartypes.c.src:4186:29: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2974     4186 |     PyLongDoubleArrType_Type.tp_print = longdoubletype_pri\n             nt;\n     2975          |                             ^\n>> 2976    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2977     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2978          |                         ^\n  >> 2979    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2980     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;",
    "nt;\n     2975          |                             ^\n  >> 2976    numpy/core/src/multiarray/scalartypes.c.src:4188:25: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2977     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2978          |                         ^\n>> 2979    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2980     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2981          |                          ^\n  >> 2982    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2983     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p",
    "2977     4188 |     PyCFloatArrType_Type.tp_print = cfloattype_print;\n     2978          |                         ^\n  >> 2979    numpy/core/src/multiarray/scalartypes.c.src:4189:26: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2980     4189 |     PyCDoubleArrType_Type.tp_print = cdoubletype_print;\n     2981          |                          ^\n>> 2982    numpy/core/src/multiarray/scalartypes.c.src:4190:30: error: 'PyTyp\n             eObject' {aka 'struct _typeobject'} has no member named 'tp_print'\n     2983     4190 |     PyCLongDoubleArrType_Type.tp_print = clongdoubletype_p\n             rint;\n     2984          |                              ^\n  >> 2985    error: Command \"/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/g\n             cc/gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -"
]